products/db-checkbook/build_scripts/summarization.py

```diff
@@ -0,0 +1,115 @@
+from pathlib import Path
+import pandas as pd
+
+from . import OUTPUT_DIR
+
+BUILD_OUTPUT_FILENAME = "historical_spend.csv"
+SUMMARY_STATS_FILENAME = "historical_spend_stats.csv"
+
+# format the display of large numbers in dataframes to use commas and 2 decimal places
+pd.options.display.float_format = "{:,.2f}".format
+
+
+def categorization_summarization_statistics(df: pd.DataFrame) -> None:
+    projects_with_geometry = df[df["has_geometry"] == True]
+
+    print("--- Value counts: 'final_category' ---")
```

Comment:
Having so many print statements feels a little noisy. It might be nice to have just one function that collects all these summary stats into either a dictionary, e.g. 
```
{'final_category': df["final_category"].value_counts() ... }
```
or a list of tuples
```
[('final_category', df["final_category"].value_counts()), ...]
```
And then you can just print that out at the end?